### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by edekeemmanuel** [community/community#8024] [Copilot Conversations]
please @Github Copilot community i have been on the waitlist for months, it seems probably i have been forgotten. please kindly help me active it, this is what i have [ERROR] [default] [2021-11-23T10:23:24.122Z] GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo ENOTFOUND api.github.com"

**Comment by davecheney** [comment]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by pedrobraz1990** [reply]
Hey there! That might be my case.... it's a corporate environment... what would be the workaround? thanks!

**Reply by davecheney** [reply]
Please keep an eye on this forum (not this thread) I believe there will be a new release announcement this week.

**Reply by pedrobraz1990** [reply]
Thanks for the prompt reply and I'm glad to hear! super excited about using it.

**Comment by edekeemmanuel** [comment]
No i don't have any of that

Sent from Yahoo Mail on Android 
 
  On Tue, 23 Nov 2021 at 12:36, Dave ***@***.***> wrote:   


Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by edekeemmanuel** [comment]
I didn't even get an activation message on my email. I don't  have any thing intercepting it

Sent from Yahoo Mail on Android 
 
  On Tue, 23 Nov 2021 at 12:36, Dave ***@***.***> wrote:   


Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by msanchezdev** [reply]
I am behind a firewall, I have been searching for the ip ranges or domain names needed to be excluded from the firewall but could not find anything.

**Comment by AngoLGW** [comment]
I set host file, writing about 140.82.114.6 api.github.com at the end

**Reply by pedrobraz1990** [reply]
thanks for the help! what is a host file? hahaha
